### DISCUSSION TITLE
Is this fork going to be maintained?

### DISCUSSION THREAD

**Discussion by redelacruz** [vscode-arduino/vscode-arduino#11] [General]
Now that the official repository is being deprecated, is this this one going to be maintained? We'd much prefer to continue developing on VSCode, so we're looking for alternatives. Currently, this repository has upstream commits that are waiting to be merged, and that's a bit concerning.

If not this one, I hope a proper community fork pops up somewhere.

**Comment by KN1HWK** [comment]
Agreed!

**Comment by lersi** [comment]
Yes, I maintain this repo, any help is appreciated.
I am a little busy with my personal life so currently I do not have much spare time.
I will make some contributions soon

**Reply by Fi1osof** [reply]
@lersi so who are you? In your profile no any projects. Is this cybersquatting? It's no looks like community project :-(

**Reply by lersi** [reply]
Hello @Fi1osof, I understand your concern.
It's weird that you did not see any project on my profile.
I created a [project](https://github.com/lersi/edk2_project_template) that makes the life easier for beginners with EDK2.
And there is another public [project](https://github.com/lersi/intersembly) of mine that enables you to run ASM instructions directly on your machine.
In addition, I made contributions to [Bus Pirate](https://github.com/DangerousPrototypes/BusPirate5-firmware) and [UEFI ReBar](https://github.com/xCuri0/ReBarUEFI) projects.

Look, I created this fork because a couple of months ago, I found a bug in the extension and like a good man, I opened an issue and also provided a pull request with a fix.
Since the maintainer ignored my issue and pull request (and also for other contributors), I saw the writing on the wall and decided to make my own fork, so these fixes could be added to an alternate version of the extension.
But this fork did not get a lot of traction, I also abandoned this fork until recently when Microsoft announced their extension deprecation, and people started to engage with this repo. Then I decided pick up the mantle and allocate the little of my spare time available to maintain this fork.

Furthermore, one of the first things that I did is to create a docker file for creating a container to build the project in, so it will be easier for others to start contributing.
And you can also see an issue where I consult with the community.

But for your question, you are right, at the moment I am the sole contributor to this fork.
I hope that other good people will start to contribute to this fork and then this will become a true community project.
It is your right to not trust this project, you can compile the project from source or even create an maintain your own fork.

Right now I see that this project is the best shot that the community has for keeping this extension alive. Better help us and contribute.

**Reply by Fi1osof** [reply]
@lersi Thank you for your detailed answer! I am glad that my personal concerns are far from the truth. All the best!

**Reply by MatthewSteeves** [reply]
@lersi Thank you for your efforts!!

**Comment by chicodaman** [comment]
Good to hear! I think the main thing needed is simple ongoing functionality, rather than new features or other labor-intensive work. As long as an Arduino sketch compiles under VSC using the Arduino CLI app it's good for my requirements....Thanks.
